forge/housekeeper/index.js

```diff
@@ -0,0 +1,55 @@
+const fp = require('fastify-plugin')
+const { scheduleTask } = require('cronosjs')
+
+/**
+ * House-keeper component
+ *
+ * Runs regular maintenance tasks to keep things clean and tidy
+ *
+ * A task is defined as an object with the properties:
+ * {
+ *    name: 'name of the task',
+ *    startup: true/false, // should the task be run whenever the platform starts
+ *    schedule: '@daily', // cron string to define schedule. '@daily' means midnight
+ *    run: async function(app) { } // the task itself
+ * }
```

Comment:
```suggestion
 * ```js
 * {
 *    name: 'name of the task',
 *    startup: true/false, // should the task be run whenever the platform starts
 *    schedule: '@daily', // cron string to define schedule. '@daily' means midnight
 *    run: async function(app) { } // the task itself
 * }
 * ```
```

Nip picking.

Turns this...
![image](https://user-images.githubusercontent.com/44235289/203100315-41f16aab-2b14-4675-81a2-e73a2ee34364.png)



into this...
![image](https://user-images.githubusercontent.com/44235289/203100277-0cdaa966-db0e-491b-b557-1940e7ac9546.png)